packages/trader/src/Modules/Trading/Components/Elements/Multiplier/cancel-deal-mobile.jsx

```diff
@@ -27,7 +27,9 @@ const DealCancellationWarning = ({
         enableApp={enableApp}
         portal_element_id='modal_root'
     >
-        <Localize i18n_default_text='Take profit and/or stop loss are not available while deal cancellation is active.' />
+        <Text size='xxxs'>
```

Comment:
Can you set the `color` prop here as well?

Reply:
noted